Release 0.1.0

better0fdead opened the pull request:
## Overview

The release introduces the `metrics-export-role` module: export metrics from `Tarantool` 3.
The release contains role for `Tarantool` 3.

## New features

* `Tarantool` 3 role with documentation.

better0fdead committed:
Release 0.1.0

Overview

    The release introduces the `metrics-export-role` module: export metrics
    from `Tarantool` 3.
    The release contains role for `Tarantool` 3.

New features

    `Tarantool` 3 role with documentation.

oleg-jukovec commented on CHANGELOG.md:
```diff
@@ -9,10 +9,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+## 0.1.0 - 2024-06-11
+
+The release introduces the `metrics-export-role` module: export metrics
+from `Tarantool` 3.
+The release contains role for `Tarantool` 3 with documentation.
```
```suggestion
from `Tarantool` 3. The release contains role for `Tarantool` 3 with
documentation.
```

oleg-jukovec reviewed (approved):
LGTM!

DifferentialOrange commented on .github/workflows/publish.yml:
```diff
@@ -14,7 +14,7 @@ jobs:
         uses: tarantool/actions/check-module-version@master
         with:
-          module-name: 'metrics-export-role'
+          module-name: 'roles.metrics-export'
```
Is it works fine? As far as I remember, action uses Tarantool 2.10

better0fdead replied:
Thx, changed.
For now reworked this job.
Will open PR in tarantool/actions to add an ability to choose tarantool version.

better0fdead committed:
CI: rewrite version-check job

Wrote our own version-check because
'tarantool/check-module-version' action installs Tarantool 2.10.
For this module we need Tarantool 3.

oleg-jukovec reviewed (commented):
```
Wrote our own version-check because 'tarantool/check-module-version' action installs Tarantool 2.10.
For this module we need Tarantool 3.
```

Please, update the commit message according the rules (72 characters per a line):

https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message

better0fdead commented:
> ```
> Wrote our own version-check because 'tarantool/check-module-version' action installs Tarantool 2.10.
> For this module we need Tarantool 3.
> ```
> 
> Please, update the commit message according the rules (72 characters per a line):
> 
> https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message

thx, changed
